test/integration/etcd/data.go

```diff
@@ -544,10 +544,10 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		// --
 	}
 
-	// k8s.io/kubernetes/pkg/apis/storage/v1beta1
+	// k8s.io/kubernetes/pkg/apis/storage/v1
 	// add csinodes if CSINodeInfo feature gate is enabled
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSINodeInfo) {
-		etcdStorageData[gvr("storage.k8s.io", "v1beta1", "csinodes")] = StorageData{
+		etcdStorageData[gvr("storage.k8s.io", "v1", "csinodes")] = StorageData{
```

Comment:
this wouldn't remove the v1beta1 fixture, it would add a v1 fixture, and change the ExpectedGVK of the v1beta1 fixture to v1